README.md

```diff
@@ -1,6 +1,6 @@
 # Dontbug Debugger
 
-Dontbug is a reverse debugger (aka _time travel debugger_) for PHP. It allows you to record the execution of PHP scripts (in command line mode or in the browser) and replay the same execution back in a PHP IDE debugger. During replay you may debug normally (forward mode debugging) or in reverse, which allows you to step over/out backwards, step backwards, run backwards, run to cursor backwards, set breakpoints in the past and so forth.
+Dontbug is a reverse debugger (aka _time travel debugger_) for PHP. It allows you to record the execution of PHP scripts (in command line mode or in the browser) and replay the same execution back in a PHP IDE debugger. During replay, you may debug normally (forward mode debugging) or in reverse, which allows you to step over/out backwards, step backwards, run backwards, run to cursor backwards, set breakpoints in the past and so forth.
```

Comment:
This is stylistic choice. Sometimes you want to have a comma to allow the users to pause but sometimes you don't need to. There is a comma after "reverse" which pauses the user. Sometimes having too many commas breaks the flow of the prose.

Thanks for the suggestion though. I may add a comma here as you've suggested but I think its OK for now...